src/app/components/home.component.ts

```diff
@@ -768,6 +768,9 @@ export class HomeComponent implements OnInit, AfterViewInit {
       if (settingsObject.remoteSettings) {
         this.remoteSettings = settingsObject.remoteSettings;
       }
+      if (settingsObject.wizardOptions) {
+        this.wizard = settingsObject.wizardOptions;
```

Comment:
After merging I'll add a few lines - maybe resetting the `wizardOptions` property `showWizard` 👍